scripts/repro_matrix/run_all_odqa.py

```diff
@@ -44,6 +44,8 @@ def print_results(metric, topics):
                         default=False, help='Skip running trec_eval.')
     parser.add_argument('--topics', choices=['triviaqa', 'naturalquestion'],
                         help='Topics to be run [triviaqa, naturalquestion]', required=True)
+    parser.add_argument('--full_topk', action='store_true',
```

Comment:
Change `--full_topk` to `--full-topk`?

More consistent with rest of code base and more readable.

Reply:
Interesting. I thought that was a parsing issue. The dash turns out to be parsed as an underscore.